CONTRIBUTING.md

```diff
@@ -43,6 +43,7 @@ Commits should follow the format <type>: <subject> or <type>(scope): <subject>
 
 Create release branch e.g `release/v1.0.0`, and push to remote.
 
+Run `lerna run build` to make sure the UMD package is up to date.
```

Comment:
@hgaitanis should it be `yarn run build`? also, you may need to specify which dir to run this command. It's not in root package.json script. Maybe just add it to project root?

Reply:
@tim-yao @hgaitanis if you use `lerna run build`, lerna will run that command in every package that supports it. So in this case you can run it from the root directory and it will do a `yarn run build` in the `Bundle` package.

Reply:
@b-schiller good to know that! Thanks for explanation.